packages/summary-table-react/documentation/SummaryTableExample.tsx

```diff
@@ -16,15 +16,50 @@ const SOFT_HYPHEN = "\u00AD";
 
 export const summaryTableExampleCode: CodeExample = `
 <SummaryTable
-    columnDescriptions={["Produkt", "Pris"]}
-    items={[
-        { label: "Person", value: "", bold: true },
-        { label: "Barne${SOFT_HYPHEN}forsikring", value: "200 kr/mnd" },
-        { label: "Behandlings${SOFT_HYPHEN}forsikring", value: "200 kr/mnd" },
-        { label: "Skade", value: "", bold: true },
-        { label: "Innboforsikring", value: "100 kr/mnd" },
-        { label: "Reiseforsikring", value: "100 kr/mnd" },
-    ]}
-    footer={{ label: "Total sum", value: "600 kr/mnd" }}
+    caption="Informasjon om produkter og pris i handlekurven"
+    header={["Produkt", "Pris"]}
+    body={
+        <tbody>
+            <tr>
+                <th scope="row">
+                    <strong>Person</strong>
```

Comment:
`<strong>` har en betydning som ikke blir helt korrekt her, mener jeg. 

> The `<strong>` element is for content that is of "strong importance," including things of great seriousness or urgency (such as warnings). ([MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong))

Et alternativ kan være å bruke `<b>`

>  [T]he `<b>` element is used to draw attention to text without indicating that it's more important

Ellers tenker jeg vi kanskje kan beholde klassen `__row--bold` og heller bruke den. Semantisk trenger vi ikke bruke `<b>` for å trekke oppmerksomheten, vi er allerede en `<th>`.

Dette er kanskje en kandidat for en komponent? `SummaryTableSectionHeader` eller noe sånt?

Reply:
Ja, var egentlig bare tilfeldig at jeg brukte strong. Men jeg synes kanskje vi burde ha en global utility klasse som setter ` font-weight: jkl.$typography-weight-bold;` istedenfor å definere det per komponent med?

Reply:
Det minte meg på: det finnes! `jkl-bold` ble lagt til i [`2837d10` (#2603)](https://github.com/fremtind/jokul/pull/2603/commits/2837d109ea94f383fabaf0ad2cbb605facc6faa9) på grunn av eksisterende "uoffisiell" bruk i for eksempel Accordion (og ExpandButton i den PRen).